### PR TITLE
Showing correct text on MFA disabled/not-enforced on Guest accounts configuration

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Showing correct text on MFA disabled/not-enforced on Guest accounts configuration.

I added the case for "not enforced" because I think makes sense.

#### Ticket Link
[MM-17211](https://mattermost.atlassian.net/browse/MM-17211)

### CODE REVIEW

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -3395,11 +3395,36 @@ export default {
...
+                        isDisabled: () => true,
+                        isHidden: it.configIsTrue('ServiceSettings', 'EnableMultifactorAuthentication'),
+                    },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'GuestAccountsSettings.EnforceMultifactorAuthentication',
+                        label: t('admin.guest_access.mfaTitle'),
+                        label_default: 'Enforce Multi-factor Authentication: ',
+                        help_text: t('admin.guest_access.mfaDescriptionMFANotEnforced'),
+                        help_text_default: '[multi-factor authentication](./mfa) is currently not enforced.',
```
Comment (jespino) [inline components/admin_console/admin_definition.jsx]: @wiersgallak I added the case where MFA is not enforced, so we can't enable MFA Enforcement for guest accounts. Is that correct? It wasn't defined in the UX.

**Comment by srkgupta** [comment]
Changes looks good. This will be tested as part of 5.16 release Guest Accounts feature. Removing the QA Review label now.

File: components/admin_console/admin_definition.jsx [hunk 2]
```diff
@@ -3395,11 +3395,36 @@ export default {
...
                             it.configIsFalse('ServiceSettings', 'EnforceMultifactorAuthentication'),
                         ),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'GuestAccountsSettings.EnforceMultifactorAuthentication',
+                        label: t('admin.guest_access.mfaTitle'),
+                        label_default: 'Enforce Multi-factor Authentication: ',
+                        help_text: t('admin.guest_access.mfaDescriptionMFANotEnabled'),
+                        help_text_default: '[multi-factor authentication](./mfa) is currently not enabled.',
```
Comment (wiersgallak) [inline components/admin_console/admin_definition.jsx]: Please capitalize Multi-factor.

File: i18n/en.json [hunk 3]
```diff
@@ -678,6 +678,8 @@
   "admin.guest_access.enableDescription": "When true, external guest can be invited to channels within teams. Please see [Permissions Schemes](../user_management/permissions/system_scheme) for which roles can invite guests.",
   "admin.guest_access.enableTitle": "Enable Guest Access: ",
   "admin.guest_access.mfaDescription": "When true, [multi-factor authentication](!https://docs.mattermost.com/deployment/auth.html) for guests is required for login. New guest users will be required to configure MFA on signup. Logged in guest users without MFA configured are redirected to the MFA setup page until configuration is complete.\n \nIf your system has guest users with login methods other than AD/LDAP and email, MFA must be enforced with the authentication provider outside of Mattermost.",
+  "admin.guest_access.mfaDescriptionMFANotEnabled": "[multi-factor authentication](./mfa) is currently not enabled.",
+  "admin.guest_access.mfaDescriptionMFANotEnforced": "[multi-factor authentication](./mfa) is currently not enforced.",
```
Comment (wiersgallak) [inline i18n/en.json]: Please capitalize Multi-factor on both of these as well.

File: components/admin_console/admin_definition.jsx (hunk 1 above)
Comment (wiersgallak) [reply components/admin_console/admin_definition.jsx]: @jespino yes that would be my expectations. Thanks for catching.

File: i18n/en.json (hunk 3 above)
Comment (jespino) [reply i18n/en.json]: Done

File: components/admin_console/admin_definition.jsx (hunk 2 above)
Comment (jespino) [reply components/admin_console/admin_definition.jsx]: Done

**Comment by jespino** [comment]
@wiersgallak I capitalized all the occurrences of "multi-factor" not only the ones related to this PR.

**Pull request merged**